caesars-palace/blocks/fragment/fragment.js

```diff
@@ -0,0 +1,39 @@
+import {
+  decorateMain,
+} from '../../scripts/scripts.js';
+
+import {
+  loadBlocks,
+} from '../../scripts/lib-franklin.js';
+
+/**
+ * Loads a fragment.
+ * @param {string} path The path to the fragment
+ * @returns {HTMLElement} The root element of the fragment
+ */
+async function loadFragment(path) {
+  if (path && path.startsWith('/')) {
+    const resp = await fetch(`${path}.plain.html`);
+    if (resp.ok) {
+      const main = document.createElement('main');
+      main.innerHTML = await resp.text();
+      decorateMain(main);
+      await loadBlocks(main);
+      return main;
+    }
+  }
+  return null;
+}
```

Comment:
If this function moved to `scripts.js` could footer use it as well? I think they are almost the same.

Reply:
We'd move the rewards code to `footer` and use the footer fragment, and it would do the trick.

Reply:
There would be some changes involved in how we handle the footer currently, looking at the code.

Reply:
https://github.com/hlxsites/caesars/issues/128